posts/2021-05-25-jakarta-ee-9-1-compatibility.adoc

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Open Liberty beta is Jakarta EE 9.1 compatible
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/jhanders34
+author_github: https://github.com/jhanders34
+seo-title: Open Liberty beta is Jakarta EE 9.1 compatible - OpenLiberty.io
+seo-description: Open Liberty beta releases have satisified the Jakarta EE 9.1 TCK and have been added to the Jakarta EE website.
+blog_description: "Open Liberty beta releases have satisified the Jakarta EE 9.1 TCK and have been added to the Jakarta EE website"
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+tags: [JakartaEE]
+---
+= Open Liberty beta is Jakarta EE 9.1 compatible
+Jared Anderson <https://github.com/jhanders34>
+:imagesdir: /
+:url-prefix: https://openliberty.io
+:url-about: /
+//Blank line here is necessary before starting the body of the post.
+
+[link=https://jakarta.ee/]
+image::img/blog/JakartaEE_Logo_Compatible-color.png[align="center"]
+
+This blog post serves as a follow-up to link:{url-prefix}/blog/2021/03/05/jakarta-ee-9-compatibility.html[my last blog post] about Open Liberty's support of Jakarta EE 9.0.  Over the last several months, the Jakarta EE community has been updating the Jakarta EE 9 TCK to enable it to use either Java SE 8 or Java SE 11.  This new version of the TCK was the bulk of the work required to complete the link:https://jakarta.ee/specifications/platform/9.1/jakarta-platform-spec-9-1.html[Jakarta 9.1 specification].  Java SE 11 support in the TCK validates that an application server using Java SE 11 is compatible with the Jakarta EE 9 specification.
+
+The Jakarta EE 9.1 specification was made available this week.  On day one of this new release, Open Liberty is one of the vendor implementations used to ratify the specification.  The Open Liberty product is compatible for both the Jakarta EE 9.1 Web Profile and Platform specifications using both Java SE 8 and Java SE 11.  What is most notable is that the version tested with the 9.1 TCK is the same version of the Open Liberty beta that was marked compatible for the Jakarta EE 9.0 specification.  As I stated in my previous blog post, Jakarta 9.1 support was anticipated due to Open Liberty's support of Java SE 11 for over two years.  
```

Comment:
What is most notable is that the version tested with the 9.1 TCK is the same version of the Open Liberty beta that was marked compatible for the Jakarta EE 9.0 specification.

--->
What is most notable is that Open Liberty was ahead of the curve for Jakarta 9.1. The Open Liberty beta version that was tested with the 9.1 TCK is the same version  that was marked compatible for the Jakarta EE 9.0 specification.

Reply:
done